layouts/codelabs/zipkin.html

```diff
@@ -84,5 +84,14 @@
     })();
   </script>
 
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-125198141-1"></script>
```

Comment:
Shouldn't these file just use the common footer?

Reply:
I am not aware of a way to use shortcodes or templates in an html file. However, this PR does introduce the snippet in to `common-footer.html`.

Reply:
Gotcha @hvent90!

@rakyll let's merge this in though in so that we can get analytics and then send a followup PR with those changes.